### PR TITLE
Improve DNS, Events, FS docs

### PR DESCRIPTION
**Pull request by hs0225**
IoT.js-DCO-1.0-Signed-off-by: Hosung Kim hs852.kim@samsung.com

### CODE REVIEW

**Comment by hs0225** [comment]
@chokobole I updated as your comments.

File: docs/api/IoT.js-API-File-System.md [hunk 1]
```diff
@@ -32,196 +30,273 @@ The following shows fs module APIs available for each platform.
...
+Renames `oldPath` to `newPath` synchronously.
 
-* `oldPath: String` - old file path
-* `newPath: String` - new file path
-* `callback: Function(err)` - callback function.
- * `err: Error`
 
-Renames `oldPath` to `newPath` asynchronously.
+### `fs.stat(path, callback)`
+* `path: String` - file path to be stated
```
Comment (chokobole) [inline docs/api/IoT.js-API-File-System.md]: please wrap type with `<>`

File: docs/api/IoT.js-API-File-System.md [hunk 2]
```diff
@@ -32,196 +30,273 @@ The following shows fs module APIs available for each platform.
...
 
-* `fd: Int` - file descriptor.
-* `buffer: Buffer` - buffer that the data will be written to.
-* `offset: Number` - offset of the buffer where to start writing.
-* `length: Number` - number of bytes to read.
-* `position: Number` - specifying where to start read data from the file, if `null`, read from current position.
+### `fs.readFileSync(path[, options])`
+* `path <String>` - file path to be opened.
+* `options <Object>` - options for the operation.
+  * `encoding <String>` Default: `null` - encoding of the file.
```
Comment (chokobole) [inline docs/api/IoT.js-API-File-System.md]: ditto

File: docs/api/IoT.js-API-File-System.md [hunk 3]
```diff
@@ -32,196 +30,273 @@ The following shows fs module APIs available for each platform.
...
+* `length <Number>` - number of bytes to read.
+* `position <Number>` - specifying where to start read data from the file, if `null`, read from current position.
+
+Reads data from the file specified by fd synchronously.
+
+
+### `fs.readFile(path[, options], callback)`
+* `path <String>` - file path to be opened.
+* `options <Object>` - options for the operation.
+  * `encoding <String>` Default: `null` - encoding of the file.
```
Comment (chokobole) [inline docs/api/IoT.js-API-File-System.md]: This is absurd. Default says there can be `null`  but type signature only allows to come as `String`

**Comment by chokobole** [comment]
LGTM

**Pull request merged**